src/services/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+import goDaddy from './goDaddy'
+const services = express.Router()
+import fetch from '../helpers/httpRequest'
+
+const getDomains = async (provider: string) => {
+  switch (provider) {
+    case 'GODADDY':
+      const { service, getDomains, options } = goDaddy
+      return fetch(`${service + getDomains}`, options)
+    default:
+      return 'Provider not supported yet'
+  }
+}
```

Comment:
I would like to split getDomains within another files but I don't know where... depends if we use more function like that.🤔

Reply:
move them into `goDaddy.ts`